ntbs-service/DataAccess/UserRepository.cs

```diff
@@ -44,6 +47,22 @@ public async Task AddUserLoginEvent(UserLoginEvent userLoginEvent)
             await _context.SaveChangesAsync();
         }
 
+        public async Task<User> GetUserByEmail(string email)
+        {
+            var user = await _context.User
+                .Include(u => u.CaseManagerTbServices)
+                .ThenInclude(c => c.TbService)
+                .ThenInclude(tb => tb.PHEC)
+                .FirstOrDefaultAsync(u => String.Equals(u.Username, email, StringComparison.CurrentCultureIgnoreCase));

```

Comment:
Does this translate to SQL properly? (I assume so, but want to be careful given the wordy equality check!

Reply:
yeap, works same as equal